test/features/deploy_kogito_build_vs_code.feature

```diff
@@ -0,0 +1,31 @@
+Feature: Deploy Kogito Build - after assets processed by VS code
+  
+
+  Background:
+    Given Namespace is created
+    And Kogito Operator is deployed
+    And Project kie-asset-re-marshaller is cloned
+    And Project kie-asset-library is cloned
+    And Project kie-asset-library is built by maven with configuration:
+      | option  | -Dactive.definitions=dmn-generated |
+      | option  | -Dactive.structures=quarkus        |
+      | native  | disabled                           |
```

Comment:
@sutaakar @radtriste hello, We would need this to be different when running community vs product, is that possible to parse and pass some value from the job env?

Reply:
Something like passing env to the maven build on behind, let's say traditional `-Dproductized` to activate a profile in kie-asset-library and we would pass that property from outside of BDD suite (like in CI job) ... is that possible?

Reply:
native enabled scenarios should have the `@native` tag so it is easy to decide which tests would run

Reply:
but this should not be into the `Background` then

Reply:
oh just understood this is not only for native ...
Then I would suggest to have different scenario outline examples with some tag on the scenario

Reply:
hm, that doesn't scale much - we then need to set kogito version to the mvn build using `-Dversion.kogito`, then the possible `-Dproductized` and there can be more property overrides needed. Especially in case of version.kogito it's not possible to hardcode the value as example.
Is there a way to promote env variables through the go run into the maven invocation? If not, is that something that would be possible to do? And could someone help with that? :-)

Reply:
This is the test script used to launch the tests: https://github.com/kiegroup/kogito-operator/blob/master/hack/run-tests.sh#L22
And the different test options in Golang are defined here: https://github.com/kiegroup/kogito-operator/blob/master/test/pkg/config/config.go

We could have a way to give the Kogito Version to Maven but this is a global setting and not specific to a scenario. So it would be applied to all the Maven commands.
`-Dproductized` is a static configuration for prod scenario or ?

I wouldn't be a big fan of having scenarios which are awaiting for specific test configuration coming via command line.
But like we have the [`native` config](https://github.com/kiegroup/kogito-operator/blob/master/test/pkg/steps/maven.go#L71) on Maven, we could have a `kogito_version` to true which would add the kogito version if it has been given as a test option. But we should not have a strong dependency on it.

Note also that specific scenario for product should be set in https://github.com/kiegroup/rhpam-kogito-operator

Reply:
I guess we could review which maven properties are needed and see how they could be integrated into the framework, that would be more efficient ;)